allensdk/test/brain_observatory/gaze_mapping/test_gaze_mapping.py

```diff
@@ -187,38 +188,38 @@ def test_pupil_position_on_monitor_in_degrees(gaze_mapper_fixture,
      np.array([[-1, 0]]),
      0),
 
-    # As if looking to upper right
+    # As if looking at upper right quadrant of screen
     ({"led_position": np.array([135, 0, 0]),
       "monitor_position": np.array([170, 0, 0]),
       "camera_position": np.array([130, 0, 0])},
-     {"cam_pupil_params": np.array([[200, 400]]),
+     {"cam_pupil_params": np.array([[200, 200]]),
       "cam_cr_params": np.array([[300, 300]])},
      np.array([[1, 1]]),
      0),
 
-    # As if looking to lower right
+    # As if looking at lower right quadrant of screen
     ({"led_position": np.array([135, 0, 0]),
       "monitor_position": np.array([170, 0, 0]),
       "camera_position": np.array([130, 0, 0])},
-     {"cam_pupil_params": np.array([[200, 200]]),
+     {"cam_pupil_params": np.array([[200, 400]]),
       "cam_cr_params": np.array([[300, 300]])},
      np.array([[1, -1]]),
      0),
 
-    # As if looking to upper left
+    # As if looking to upper right quadrant of screen
```

Comment:
Nitpicky but all the other ones were changed
```suggestion
    # As if looking at upper right quadrant of screen
```